content/docs/12-enterprise-version/06-air-gap-repo.md

```diff
@@ -1,108 +1,731 @@
+
+10. Click on **Next** to complete the deployment wizard. Upon completion, the cloning process will begin. The cloning process takes a few minutes to complete.
+
+
+11. Power on the VM and click on the **Launch Web Console** button to access the instance's terminal.
+
+
+12. Configure a static IP address on the node by editing **/etc/netplan/50-cloud-init.yaml**. 
+
+  <br />
+
+  ```shell
+  sudo vi /etc/netplan/50-cloud-init.yaml
+  ```
+
+  Use the following sample configuration as a starting point but feel free to change the configuration file as required for your environment. To learn more about Netplan, check out the [Netplan configuration examples](https://netplan.io/examples) from Canonical.
+
+  <br />
+
+  ```yaml
+  network:
+    version: 2
+    renderer: networkd
+    ethernets:
+      ens192:
+        dhcp4: false
+        addresses:
+          - 10.10.244.9/18 # your static IP and subnet mask
+        gateway4: 10.10.192.1 # your gateway IP
+        nameservers:
+            addresses: [10.10.128.8] # your DNS nameserver IP address.
+  ```
+
+  To exit Vi, press the **ESC** key and type `:wq` followed by the **Enter** key. <br /> <br />
+
+13. Issue the `netplan` command to update the network settings.  
+
+  <br />
+
+  ```shell
+  sudo netplan apply
+  ```
+
+14. Give the instance one to two minutes before issuing the following command. The next step is to start the airgap setup script that stands up the Spectro Repository. Issue the command below and replace `X.X.X.X` with the static IP you provided to the Netplan configuration file.
+
+  <br />
+
+  ```shell
+  sudo /bin/airgap-setup.sh X.X.X.X
+  ```
+
+  Record the output of the setup command as you will use it when deploying the Quick Start appliance later on in the installation process.
+
+  Example Output: 
+  ```shell
+  Setting up Manifests
+  Setting up Manifests
+  Setting up SSL Certs
+  Setup Completed
+
+  Details:
+    -------
+  Spectro Cloud Repository
+  UserName: spectro
+  Password: admin@airgap
+  Location: https://10.10.249.12
+  Artifact Repo Certificate:
+  LS0tLS1CRUdJ.............<condensed>
+
+  Pack Registry
+  URL: https://10.10.249.12:5000
+  Username: admin
+  Password: admin@airgap
+  ```
+
+15. If you need to configure the instance with proxy settings, go ahead and do so now. You can configure proxy settings by using environment variables. Replace the values with your environment's respective values.
+
+  <br />
+
+  ```shell
+  export http_proxy=http://10.1.1.1:8888
+  export https_proxy=https://10.1.1.1:8888
+  export no_proxy=.example.dev,10.0.0.0/8
+  ```
+
+16. The next set of steps will download the required binaries to support a Palette installation, such as the Palette Installer, required Kubernetes packages, and kubeadm pages. You can download these artifacts from the instance, or externally and transfer them to the instance. Click on each tab for further guidance.
```

Comment:
Kubernetes packages, and kubeadm packages.

Reply:
Nice catch